modules/spring-data-2.0/src/main/java/org/apache/ignite/springdata20/repository/support/IgniteRepositoryFactory.java

```diff
@@ -107,11 +118,38 @@ public IgniteRepositoryFactory(String springCfgPath) {
 
         Assert.hasText(annotation.cacheName(), "Set a name of an Apache Ignite cache using @RepositoryConfig " +
             "annotation to map this repository to the underlying cache.");
+        String cacheName = null;
+        if (!looksLikeExpression(annotation.cacheName())) {
+            cacheName = annotation.cacheName();
```

Comment:
One line if statements should be without brackets:

```java
if (!looksLikeExpression(annotation.cacheName()))
     cacheName = annotation.cacheName();
```

Reply:
fixed